test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -516,11 +516,10 @@ foo
 Describe "WindowStyle argument" -Tag Feature {
     BeforeAll {
         $defaultParamValues = $PSDefaultParameterValues.Clone()
```

Comment:
Since `$PSDefaultParameterValues` is no longer needed, this line can be removed then.
Same to the `AfterAll` block.